src/intl/en/page-upgrades-index.json

```diff
@@ -43,9 +43,9 @@
   "page-upgrades-dive": "Dive into the vision",
   "page-upgrades-dive-desc": "How are we going to make Ethereum more scalable, secure, and sustainable? All while keeping Ethereum's core ethos of decentralization.",
   "page-upgrades-docking": "The Merge",
-  "page-upgrades-merge-answer-1": "The Merge is when Mainnet begins using the Beacon Chain for consensus, and proof-of-work is turned off, coming soon.",
+  "page-upgrades-merge-answer-1": "The Merge occurred via the Paris upgrade on September 15, 2022.",
```

Comment:
Don't love this phrasing but not blocking, and I can't think of anything better off the top of my head.